### ISSUE TITLE
why dwa didn't consider common scale of gradient?

### ISSUE THREAD

**Issue by LMerCy** [lorenmt/mtan#54]
It seems dwa only considers the rate of change of loss for each task. without the common scale of gradient, which introduced by GradNorm. So can dwa get comapareble results to GradNorm?

**Comment by lorenmt** [comment]
At the time of submission, GradNorm did not have an open-sourced implementation. You may check out some survey papers which compare DWA with GradNorm along with other optimisation methods. https://arxiv.org/pdf/2004.13379.pdf The performance typically depends on the choice of task.

Besides, we have recently submitted a new optimisation method which is currently under review. Stay tuned. We may release it to arxiv very soon.

**Comment by LMerCy** [comment]
@lorenmt Thanks for your quick response. I'm looking forward your new work!

**Comment by lorenmt** [comment]
If you are still interested in this research, please check out our new work: https://shikun.io/projects/auto-lambda. :)

**Comment by LMerCy** [comment]
> If you are still interested in this research, please check out our new work: https://shikun.io/projects/auto-lambda. :)

Great work, thanks! Have you ever try to use these work in small network such as mobilenet0.25 in detection and segmentation task,  i tried uncertainty loss in this case, it doesn't work.

**Comment by lorenmt** [comment]
No, I haven't tried experimenting with these tasks. Were you working on COCO or other large-scale experiments?

**Comment by LMerCy** [comment]
@lorenmt Not coco, our own dataset, but larger than coco.

**Comment by lorenmt** [comment]
My personal experience is large-scale dataset would not have a strong improvement with multi-task learning. Since a good representation quality can be achieved solely with task-specific training and a lot of training data. i.e. the network is not easy to overfit.
  

  
I don’t know… I will work on large-scale datasets this upcoming internship. Hopefully to have more insights on these problems.
  
  
  
  
  
  
  

  
  
>

**Comment by LMerCy** [comment]
@lorenmt Thanks, i agree a good representation is needed.Maybe i can try self-supervised traning because we have many unlabeled data.
